App.tsx

```diff
@@ -5,25 +5,38 @@ import theme from './src/themes';
 import Cryptos from './src/screens/Cryptos';
 import {NavigationContainer} from '@react-navigation/native';
 import {createStackNavigator} from '@react-navigation/stack';
+import AddCryptoScreen from './src/screens/AddCryptoScreen';
+import {Provider} from 'react-redux';
+import {store} from './src/redux/store';
+import 'react-native-gesture-handler';
 
 const stack = createStackNavigator();
 
 const App = () => (
-  <>
-    <ThemeProvider theme={theme}>
-      <NavigationContainer>
-        <stack.Navigator initialRouteName="crypto">
-          <stack.Screen
-            name="crypto"
-            component={Cryptos}
-            options={{
-              headerShown: false,
-            }}
-          />
-        </stack.Navigator>
-      </NavigationContainer>
-    </ThemeProvider>
-  </>
+  <Provider store={store}>
+    <>
+      <ThemeProvider theme={theme}>
+        <NavigationContainer>
+          <stack.Navigator initialRouteName="crypto">
+            <stack.Screen
+              name="crypto"
+              component={Cryptos}
+              options={{
+                headerShown: false,
```

Comment:
move this to the options of the navigator, instead of the options of all the screens